course-mw/sunbird-util/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraDACImpl.java

```diff
@@ -23,7 +22,59 @@
 import org.sunbird.common.responsecode.ResponseCode;
 
 public class CassandraDACImpl extends CassandraOperationImpl {
-    
+
+
+  @Override
+  public Response getBlobAsText(RequestContext requestContext, String keySpace, String table, Map<String, Object> filters, List<String> properties) {
+    Response response = new Response();
+    Session session = connectionManager.getSession(keySpace);
+
+    StringBuilder sb = new StringBuilder();
+    if (null != properties && !properties.isEmpty()) {
+      sb.append("select contentid, ");
+      StringBuilder selectFields = new StringBuilder();
+      for (String property : properties) {
+        selectFields.append("blobAsText(").append(property).append(") as " ).append(property);
+        selectFields.append(", ");
+      }
+      sb.append(StringUtils.removeEnd(selectFields.toString(), ", "));
+      sb.append(" from " + keySpace + "." + table );
+    }
+
+    StringBuilder where = new StringBuilder();
+    if (null!= filters && !filters.isEmpty()) {
+      where.append(" where ");
+      for (Map.Entry<String, Object> filter : filters.entrySet()) {
+        Object value = filter.getValue();
+        if (value instanceof List)
+          where.append(filter.getKey()).append(" in ").append("?");
+        else
+          where.append(filter.getKey()).append(" = ").append("?");
+
+        where.append(" and ");
+      }
+      sb.append(StringUtils.removeEnd(where.toString(), " and "));
+    }
+
+    PreparedStatement ps = session.prepare(sb.toString());
+    BoundStatement bound = ps.bind(filters.values().toArray());
+
+    logger.debug(requestContext, ps.getQueryString());
+
+    try {
+      ResultSet results = session.execute(bound);
+      response = CassandraUtil.createResponse(results);
+
+    } catch (Exception e) {
+      ProjectLogger.log(Constants.EXCEPTION_MSG_FETCH + table + " : " + e.getMessage(), e);
```

Comment:
@pritha-tarento : its a good practice to have a different message for every method. The log message is same as the one used in other methods in this class.

Reply:
addressed the log message for this.